azure-pipelines.yml

```diff
@@ -91,6 +91,10 @@ jobs:
   - bash: ci/scripts/clang-format.sh $SYSTEM_PULLREQUEST_TARGETBRANCH
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: clang-format (C/C++ lint)
+    # Validate testplans with schema (json schema)
+  - bash: ci/scripts/testplans-validate.sh
+    condition: eq(variables['Build.Reason'], 'PullRequest')
+    displayName: Validate testplans with schema
```

Comment:
@jwnrt recently added quick lint for GitHub CI. Maybe this job should be added there instead?

Reply:
Sure, this change was before that was added, I'm happy to look at moving this to the Github Actions CI instead.

(Ironically, I made a Github Actions workflow the first time I made this PR, but then changed it to an Azure job as we weren't using Github Actions for the quick lint job at the time :sweat_smile:)